src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -531,6 +534,17 @@ public static String calcMD5(final byte[] bytes) {
     public static String calculateFileMD5( final File file ) throws IOException{
         return Utils.calcMD5(FileUtils.readFileToByteArray(file));
     }
+    public static String calculatePathMD5(final Path path) throws IOException{
```

Comment:
Add javadoc for the new overload of `calculatePathMD5()`

Reply:
Also, can you make sure that these new method overloads are covered by unit tests?

Reply:
I added Javadoc and made sure that the first calls the second. Since the first is tested, that also covers the second automatically.